pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,33 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// RadosNamespaceMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type RadosNamespaceMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemoteNamespace is the name of the radosNamespace on secondary cluster pool
```

Comment:
```suggestion
        // RemoteNamespace is the name of the CephBlockPoolRadosNamespace on the secondary cluster CephBlockPool
```
How about above suggestion as we need to represent the CR's here